src/styles/mediaqueries.scss

```diff
@@ -48,10 +77,30 @@ For Tablet
     }
   .mission {
     .missionTitle {
-      font-size: 1rem;
+      font-size: 1.25rem;
     }
     .missionDetails {
-      font-size: .8rem;
+      font-size: 1rem;
+    }
+  }
+  .broCol{
+    .stripes {
+      width: 70%;
+      margin: -1rem 0 10rem 6rem;
```

Comment:
Ya again, do position: absolute on the inner elements and position to their spot instead of using margins like this.